### ISSUE TITLE
Split up .csv file into training set and cross validation set

### ISSUE THREAD

**Issue by abarendregt** [jdzak/bike-sharing-demand#1]
To best optimize the data, it should be split by (approx) 70% training data and 30% cross validation data.  A feature in Octave is needed to randomly split the .csv file into these two groups.  

Note:  two separate files should be saved, or the way they are split 'randomly' should be repeatable so that we'll know if our changes make the Cost Functions and gradient decent better or worse
